assignments/assignment7/faulty-oops.md

```diff
@@ -0,0 +1,249 @@
+# Fault tracing
```

Comment:
@rajatchaple appreciate the markdown here, but typically you'd use code blocks for the copy/paste data and make your text use default formatting.